### ISSUE TITLE
LogTruncateSize validation issue

### ISSUE THREAD

**Issue by richardhauer** [PowerShell/xWebAdministration#260]
It appears that validation on the xIisLogging resource's LogTruncateSize parameter is being handled incorrectly.  The parameter (and its accompanying validation) is typed as a string.  Consequently, the value `536,870,912` (passed as an int, but coerced into a string) fails validation as it is outside the range `'1,048,576'` to `'4,294,967,295'`.  This is clearly not the case, were the values being handled as integers as they should be.  Handled as strings, the leading `5` is indeed bigger than the `1` to `4` that the range specifies.

**Comment by remcoeissing** [comment]
I looked at this issue and indeed the current validation will prevent values starting with 5 till 9 from being configured. This issue also applies to xWebsite, which has the same validation logic for LogTruncateSize.

I made a change in my local repository that resolves the validation issue. I'm considering how best to handle the mof schema. Changing the type from `String` to `UInt64` will break all existing configurations.
